### PR TITLE
completed css-canvas-medical exercise. - Steve Behm 

### PR DESCRIPTION
**Pull request by SteveBehm**
Screen Shot 2022-01-30 at 1 30 02 PM
Screen Shot 2022-01-30 at 1 30 29 PM

### CODE REVIEW

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.hero-heading {
+  color: rgb(255 255 255);
+  font-family: "Tiempos Headline", sans-serif;
+  padding-left: 60px;
+  padding-right: 400px;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:76]: Why is this necessary?

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+  padding-top: 50px;
+}
+
+.hero-para {
+  color: rgb(255 255 255);
+  font-family: Avenir, sans-serif;
+  padding-left: 60px;
+  padding-right: 425px;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:84]: Why is this necessary?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-canvas-medical/styles.css (hunk 1 above)
Comment (SteveBehm) [reply css-canvas-medical/styles.css:76]: is my padding bad practice? I the font color and style is there because the heading and paragraph have different fonts. Can you give me a little more direction with what you are trying to have me notice?

I am guessing there is a better way to do the alignment of the text?

File: css-canvas-medical/styles.css (hunk 2 above)
Comment (SteveBehm) [reply css-canvas-medical/styles.css:84]: Screen Shot 2022-02-05 at 2 58 47 PM
Screen Shot 2022-02-05 at 2 59 42 PM


Is this the way you'd prefer to see the indentation that then follows from row to row?

**Review by codingcodymiller — APPROVED** [review]
Much better!
